### PR TITLE
condiment rework

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
- reclaim condiment packets for plastic, bottles and shakers for glass
- refactors condiments to be better
- remove unused prototypes
- remove (mostly unused) small condiment bottles, made vinegar a 30u bottle instead

**Media**
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:
- tweak: Chef's vinegar bottles now hold 30u instead of 15u.

### CODE REVIEW

**Comment by deltanedas** [comment]
still have to fix the funny test

**Comment by deltanedas** [comment]
ok @metalgearsloth i found out why it fails, TrashOnEmptyComponent is messing with tags on component startup which breaks things, how should i fix

File: Resources/Maps/bagel.yml [hunk 1]
```diff
@@ -76381,7 +76381,7 @@ entities:
       type: Transform
     - tags: []
       type: Tag
-- proto: FoodCondimentBottleSmallHotsauce
```
Comment (metalgearsloth) [inline Resources/Maps/bagel.yml:76384]: Prototype renames need an entry in migration.yml

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Resources/Maps/bagel.yml (hunk 1 above)
Comment (deltanedas) [reply Resources/Maps/bagel.yml:76384]: would i revert bagel change too then

Comment (metalgearsloth) [reply Resources/Maps/bagel.yml:76384]: Yeah or emisse is going to murder you

Comment (deltanedas) [reply Resources/Maps/bagel.yml:76384]: okey

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**